### PR TITLE
Fix header

### PR DESCRIPTION
**Pull request by jcenturion**
screen shot 2016-05-02 at 12 38 46 pm

### CODE REVIEW

File: build/bundle.js [hunk 1]
```diff
@@ -134,13 +163,35 @@ var logsTemplate = s(function () {/*
         .a0-logs-lines {
             min-height: 100% !important;
         }
+
+        header.dashboard-header nav li .btn-dro i {
+          top: 21px;
+        }
     </style>
+    <script type="text/javascript">
+      sessionStorage.setItem('token', '<%- a0Token %>');
```
Comment (tjanczuk) [inline build/bundle.js]: This is a SPA that does not use the a0Token at all. Why do we bother shipping it to the browser only to store and later remove it from session storage?

Comment (tjanczuk) [inline build/bundle.js]: Or does it use to display the nice picture?

**Comment by vctrfrnndz** [comment]
@jcenturion Is it possible to fit the "terminal" inside a standard container (like the header) and add the standard breadcrumb like other extensions?

File: build/bundle.js (hunk 1 above)
Comment (jcenturion) [inline build/bundle.js]: The navbar component uses the token for rendering the picture.

**Comment by jcenturion** [comment]
Sure. I will modify it.

**Comment by jcenturion** [comment]
@vctrfrnndz:

screen shot 2016-05-02 at 6 18 17 pm

**Comment by vctrfrnndz** [comment]
@jcenturion Looks great, I think this is just missing the name of the extension before the therminal, like this example:
image

**Comment by jcenturion** [comment]
screen shot 2016-05-04 at 11 11 49 pm

**Comment by vctrfrnndz** [comment]
👌

File: build/bundle.js [hunk 2]
```diff
@@ -122,8 +151,6 @@ var logsTemplate = s(function () {/*
           top: 0;
         }
         .logs {
-          position: absolute;
```
Comment (ggoodman) [inline build/bundle.js]: How will the logs container know where to position itself?

File: build/bundle.js [hunk 3]
```diff
@@ -108,8 +129,16 @@ var logsTemplate = s(function () {/*
     <meta charset="utf-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <link rel="shortcut icon" href="https://cdn.auth0.com/styleguide/2.0.1/lib/logos/img/favicon.png" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.auth0.com/manage/v0.3.973/css/index.min.css" />
+    <link rel="stylesheet" type="text/css" href="https://cdn.auth0.com/styleguide/3.1.6/index.css">
     <script src="https://cdn.auth0.com/webtaskWidget/auth0-webtask-widget-1.min.js"></script>
```
Comment (ggoodman) [inline build/bundle.js]: Update to `v2.0`.

**Pull request merged**